fastlane_core/lib/fastlane_core/core_ext/string.rb

```diff
@@ -0,0 +1,13 @@
+class String
```

Comment:
This is copied into `fastlane_core` for now, but we will remove it from `fastlane` as part of a follow-up PR